supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/api/request/BatchRequestExecutor.scala

```diff
@@ -144,7 +144,9 @@ object BatchRequestExecutor {
       // Get instances that can be created with this AllocationPolicy, only instances[0] is supported
       val instancePolicy = allocationPolicy.getInstances(0).getPolicy
       val machineType = instancePolicy.getMachineType
-      val preemptible = instancePolicy.getProvisioningModelValue == ProvisioningModel.PREEMPTIBLE.getNumber
+
+      // SPOT VM is used as preemptible VM instances in Batch
+      val preemptible = instancePolicy.getProvisioningModelValue == ProvisioningModel.SPOT.getNumber
```

Comment:
I'm wondering if it would be safer to check for either - I think the word "preemptible" is used in the cost catalog.